src/bio/bio_context.c

```diff
@@ -66,6 +75,32 @@ blob_cp_arg_fini(struct blob_cp_arg *ba)
 	ABT_mutex_free(&ba->bca_mutex);
 }
 
+static void
+free_blob_msg_arg(struct blob_msg_arg *bma)
```

Comment:
minor, seems blob_msg_arg_free/_alloc() is more align with naming style

Reply:
Ok.